apps/galaxy/src/modules/auth/SignInForm/index.tsx

```diff
@@ -0,0 +1,30 @@
+import SignInForm from "./SignInForm";
+import { Flex, Image, Text } from "@cc/ui-chakra";
+
+const SignIn = () => {
+  return (
+    <Flex height="calc(100vh - 30px)" justifyContent="center" alignItems="center">
+      <Flex width="500px" flexDir="column" alignItems="center">
+        <Flex flexDir="column" alignItems="center">
```

Comment:
Should we decouple this into a `<Logo>` component?

Reply:
I changed it to only use an `<Image>`  now, and I don't think we need to decouple an image into a component